class_generator/README.md

```diff
@@ -70,6 +70,15 @@ class-generator --kind Pod --add-tests
 uv tool install --python python3.9 openapi2jsonschema
 ```
 
+If install fail ([Issue 1455](https://github.com/astral-sh/uv/issues/1455)) try to install manually:
+
+```bash
+git clone https://github.com/instrumenta/openapi2jsonschema
+cd openapi2jsonschema
+sed -i s/'pyyaml = "^5.1"'/'pyyaml = ">=6.0"'/g  pyproject.toml
```

Comment:
```
sed -i .bk s/'pyyaml = "^5.1"'/'pyyaml = ">=6.0"'/g  pyproject.toml
```
On MacOS I had to specify an extension for the command to work correctly and save a backup copy of the file, otherwise it works for me too